src/main/java/org/testeditor/fixture/host/s3270/actions/ControlCommand.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2017 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.fixture.host.s3270.actions;
+
+import org.testeditor.fixture.host.exceptions.CommandNotFoundException;
+
+/**
+ * Following special commands can be used in tests :
+ * <ul>
+ * <li>{@link #BACKTAB} - move cursor to previous input field</li>
+ * <li>{@link #CLEAR} - clear screen</li>
+ * <li>{@link #DELETE} - delete character under cursor</li>
+ * <li>{@link #ENTER} - send an command</li>
+ * <li>{@link #ERASE_EOF} - erase to end of current field</li>
+ * <li>{@link #PF3} - get back to to previous screen</li>
+ * <li>{@link #RESET} - reset locked keyboard</li>
+ * <li>{@link #TAB} - move cursor to next input field</li>
+ * </ul>
+ * <p>
+ * These commands are a subset of the complete acion list.
+ * <p>
+ * See <a href= "http://x3270.bgp.nu/s3270-man.html#Actions" target=
+ * "_top">http://x3270.bgp.nu/s3270-man.html#Actions</a> for a complete list of
+ * actions.
+ *
+ */
+public enum ControlCommand {
+
+    BACKTAB("BackTab"), CLEAR("Clear"), DELETE("Delete"), ENTER("Enter"), ERASE_EOF("EraseEOF"), PF3("PF(3)"), RESET(
+            "Reset"), TAB("Tab");
+    private String command;
+
+    private ControlCommand(String state) {
+        this.command = state;
+    }
+
+    public String getCommand() {
+        return command;
+    }
+
+    public static ControlCommand getControlCommand(String input) {
+        ControlCommand[] values = ControlCommand.values();
```

Comment:
this method seems to be repeated for every enum. could it be extracted into a generic utility method? does a respective utility method exist (e.g. in some apache common module)?